### PR TITLE
Table filter component

### PR DESCRIPTION
**Pull request by urchykli**
I added the styles for the filter and reset filter buttons.  Serven will add functionality for the reset button.  Also, per our conversation, I changed the .btn:focus class to use box shadow instead of outline.  Please let me know what I need to update.  Thanks!

### CODE REVIEW

**Comment by urchykli** [comment]
Closes #55

**Comment by urchykli** [comment]
I made the requested changes, but I had to override the hover text color on the reset button when I added btn--transparent-dark.  Let me know if I missed anything.  Thanks!

File: src/scss/components/_filter-button.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  }
+
+  &:focus:hover {
+    box-shadow: 0 0 0 2px #85b7db;
+  }
+}
+
+.filter__reset {
+  &:focus {
+    box-shadow: none;
```
Comment (jnschrag) [inline src/scss/components/_filter-button.scss]: The reset button should have a box-shadow on focus. Right now, there's no way to tell if the reset button is focused when navigating via a keyboard.

File: src/scss/components/_filter-button.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  }
+
+  &:hover {
+    color: $color__white-160 !important;
```
Comment (jnschrag) [inline src/scss/components/_filter-button.scss]: I checked this with Tucker when I was making the button components - the reset button should have white text on hover. This override can be removed.

**Comment by urchykli** [comment]
These changes have been made.  Thanks!

**Pull request merged**